sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -669,3 +669,7 @@ CREATE TABLE `api_token` (
 ) ENGINE = InnoDB;
 #EndIf
 
+#IfMissingColumn users patient_menu_role
+ALTER TABLE `facility` ADD `mail_street` VARCHAR(30) NOT NULL AFTER `facility_taxonomy`, ADD `mail_city` VARCHAR(35) NOT NULL AFTER `mail_street`, ADD `mail_state` VARCHAR(2) NOT NULL AFTER `mail_city`, ADD `mail_zip` VARCHAR(10) NOT NULL AFTER `mail_state`;
+ALTER TABLE `facility` ADD `oid` INT(15) NOT NULL DEFAULT '0' FIRST;
```

Comment:
@sjpadgett ,
Can I get clarification on this. saw it discussed in the forum. Does this need to be set in the facility form? Is it only numbers?
thanks, -brady

Reply:
@juggernautsei ,
Likely no need for FIRST here, but lets wait for Jerry's input.

Reply:
Yes, he did not clarify and I just left it int

Reply:
Thanks @sjpadgett ,
@juggernautsei , so sounds like no need for FIRST; ie. it can go at the end. And will need to allow user to enter in this value in the facilities form.

Reply:
Ok, I can update the form to add the OID entry. I will post an update tomorrow.